assets/javascripts/discourse/templates/components/minimal-gamification-leaderboard-row.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="user {{if rank.isCurrentUser "user-highlight"}}" id="leaderboard-user-{{rank.id}}">
```

Comment:
the "user" class is too generic IMO. Maybe we use something like "compact-leaderboard-user"?

Reply:
Ah, I guess it is following the same styles/classes as the main leaderboard? May be OK in that case.